galaxy_ng/app/access_control/access_policy.py

```diff
@@ -218,10 +218,6 @@ class AppRootAccessPolicy(AccessPolicyBase):
     NAME = "AppRootViewSet"
 
 
-class NamespaceAccessPolicy(AccessPolicyBase):
-    NAME = "NamespaceViewSet"
```

Comment:
We need to apply our existing access policies to the new pulp namespace

Reply:
If the pulp_ansible namespace apis provide an equivalent but different and incompatible API, almost anything using them currently will need to be updated.

Do we want to provide any sort of compatibility layer ?

I've been working on the expectation we are attempting to avoid implementing any new API in galaxy_ng. (ie, no subclassing of pulp_ansible viewsets etc). Is that still a valid assumption?